src/server/services/generate-arpa-report.js

```diff
@@ -37,7 +43,7 @@ async function loadTemplate (templateName) {
   }
 
   const worksheet = workbook.Sheets[workbook.SheetNames[0]]
-  return XLSX.utils.sheet_to_json(worksheet, { header: 1 })
+  return XLSX.utils.sheet_to_json(worksheet, { header: 1, blankrows: false })
```

Comment:
i remember you saying that this sometimes messes with things, like if we expect a specific cell at a specific index and then the indexing is thrown off after blank rows are removed. should this be the default?

Reply:
It seems that the xlsx export template files have 999 rows, mostly empty.  If we don't strip empty rows, then any new data we append will start at row 1000 — I think that's likely to be confusing to either humans or the treasury portal.

The issue I saw in the past, was with blank row elimination during read of input files.  We could try the inverse setting on the file parser for input sheets; that's not a bad idea.  👍